.internal-ci/helm/consensus-node-config/values.yaml

```diff
@@ -59,3 +59,9 @@ global:
     # Add signed tokens.json with --set-file=global.node.tokensConfig.tokensSignedJson=tokens.signed.json
     # tokensConfig:
     #   tokensSignedJson: |-
+
+  ### Enable haproxy IP blacklist for ingress
+  # pattern is the object in the configmap shared between infra-haproxy-blacklist and haproxy kubernetes-ingress
```

Comment:
:question: Where is this pattern? I did some quick grepping of `infra-haproxy-blacklist` the mobilecoin repo and the github orgs but came up empty

Reply:
The configmap name is defined on when you launch the haproxy, that's an internal repo that has all of those definitions.  The `data` object will be called `blocked-countries`.  This object will be a newline separated list of CIDRs to be blocked. The list is generated by on the scripts/deployment in `infra-haproxy-blacklist`.